specs/core/0_beacon-chain.md

```diff
@@ -1587,41 +1608,36 @@ _Note_: If there are skipped slots between a block and its parent block, run the
 
 ### Per-slot processing
 
-Below are the processing steps that happen at every slot.
-
-#### Slot
+Below are the processing steps that happen at every `slot >= GENESIS_SLOT`.
 
+* Set `state.latest_state_roots[state.slot % SLOTS_PER_BATCHING] = hash_tree_root(state)`.
```

Comment:
This means that the `latest_state_roots` for a `slot` is the post state of `slot - 1` and the _pre_ state of `slot`.
Is this the intention?

Reply:
It seems a bit off on the expectation of looking up a `state` at `slot`

Reply:
We discussed this yesterday. Notice that `slot += 1` isn't yet applied, so all is good :)